Lab#1.ipynb

```diff
@@ -0,0 +1,283 @@
+      "text/plain": [
+       "<Figure size 1440x1080 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#resize plot\n",
+    "plt.rcParams[\"figure.figsize\"] = (20,15)\n",
+    "fig, ax = plt.subplots(1,1)\n",
+    "plt.tick_params(labelsize = 24)\n",
+    "\n",
+    "d = stats.expon.rvs(loc = 0., scale = 2, size = 100000)\n",
+    "ax.hist(d, 50, density=True)\n",
+    "\n",
+    "x = np.linspace(0.05, 10.05, 2000)\n",
+    "plt.xlim([0.05, 10.05])\n",
+    "ax.plot(x, stats.expon.pdf(x, loc=0, scale=2), 'r-', lw=3, alpha=0.8, label='expon pdf')\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Suppose we have a radioactive sample (plotted and standardized) that has a measured burst of gamma rays after 4 years (located at x = 4); What is the probability that what we observed at the end of 4 years was produced the background rather than a signal?\n",
+    "\n",
+    "In the form of integrals, this question becomes:\\\n",
+    "$$P = \\int_4^\\infty f(x) \\,dx= F(\\infty) - F(4) = 1 - F(4)$$\n",
+    "\n",
+    "where $f(x)$ is the exponential pdf and $F(x)$ is the exponential cdf, and $x$ is the time in years since the start of the experiment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 143,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.1353352832366127"
+      ]
+     },
+     "execution_count": 143,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "probability = 1 - stats.expon.cdf(4, loc=0, scale=2)\n",
+    "probability"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we compare the probability that the background produced the 'signal' to the probability starting from the right side of the normal distribution to find our certainty in terms of $\\sigma$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 144,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1.1015196284987503"
+      ]
+     },
+     "execution_count": 144,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "stats.norm.ppf(1 - probability, loc=0, scale=1)"
```

Comment:
Should this scale be different compared to what you used to calculate the probability?